service.games.discord.richpresence/default.py

```diff
@@ -0,0 +1,37 @@
+import xbmc
```

Comment:
usually it's a good practice to keep this file (as the main entrypoint) as small as possible. Reason is the fact this is the only file that is not compiled to bytecode.
Can you move this to `resources/lib/richpresence.py` and in this file simply import and run?

```
from resources.lib.richpresence import mystuff

mystuff.run()
```

There's a warning for this in addon checker